Winapp3/Archived entries.ini

```diff
@@ -1,12 +1,15 @@
 ; Version: 191025
-; # of archived entries: 26
-;
-; Archived entries.ini is fully licensed under the CC-BY-SA-4.0 license agreement. Please refer to our license agreement before using Archived entries.ini: https://github.com/MoscaDotTo/Winapp2/blob/master/Winapp3/License.md
-; If you plan on modifying, distributing, and/or hosting Archived entries.ini for your own program or website, please ask first.
-;
-; This file contains entries that were removed from the main Winapp2.ini file because they are mainly intended for discontinued or outdated programs.
-; You should NOT use this file as it is for archiving purposes only.
+; # of entries: 39
+; 
+; Winapp2.ini is fully licensed under the CC-BY-SA-4.0 license agreement. Please refer to our license agreement before using Winapp2: https://github.com/MoscaDotTo/Winapp2/blob/master/License.md
```

Comment:
Archived entries.ini

Reply:
Sorry I had used the Debug Tool and forgot to change back the Header.


As for the entries, which have little to less impact, my statement is following:

Downloads Folder Cleaning is supported in Windows Clean Manager also, so it's alright to add it here.
Downloads Part File Cleaning is even very safe to use in most cases.
Downloads Exe and Msi Cleaning is even more safe to use, since most of them are Installer files in General.

As for Audio, Image and Video file type searcher, they should not be added to Winapp3.ini, since they are only for specific searches, which some power users might use and this would help most of the users not to have to write the entry themself.

As for the Infile Cleaning Rules, this is made from me to start a debate in the Community to support Bleachbit, SystemNinja to support these in future, so we can clean the most private entries. The last resort for me was to add them here, so in future developers copy and paste them into their own winapp2.ini files. Perhaps CCleaner could in future support also infile cleaning, which is the most important part.

Reply:
![Image](https://user-images.githubusercontent.com/57820130/68997811-c91ffa00-08aa-11ea-938a-d99758fe86e7.jpg)

Reply:
As @ROCKNROLLKID has already written, this file is only for entries removed from the **official** Winapp2.ini, because the corresponding programs are outdated or no longer functional. The pull request can therefore be closed.

(BTW Microsoft plans to remove "Downloads" with the next feature upgrade.)

Reply:
Okay, you are right.